tests/tests-mac.sln

```diff
@@ -11,6 +11,8 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "GuiUnit_xammac_mobile", "..
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "apitest-unified", "apitest\apitest-unified.csproj", "{7A0EDA95-30A6-43E1-AD43-368AD95AC48B}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "xammac_tests", "xammac_tests\xammac_tests.csproj", "{22BF080C-AFAD-445B-8BB5-42B9E7FDBC68}"
+EndProject
```

Comment:
Yah I commented on rolf/chris's comments that I figured out I had the wrong project type so it was using system mono.  Fixing the project and using mobile as it was supposed to fixed those issues.

I could dfeinitely add it to apitest instead of a new project.  The one thing there is that right now it won't work for classic (lotta importws have to be fixed with #ifs to use MonoMac.Foo instead of  MonoTouch.Foo to get classic working